### ISSUE TITLE
[Accessibility]A11y_ AzureDataStudio_ NotebookQueryTable_ScreenReader:Screen reader is not announcing the table name.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5615]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
 GitHubTags: #A11yMAS; #A11yTCS; #MAS1.31;#MAS4.12 #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
NVDA Version: 2019.1
JAWS Version: 2019

**Additional Details:**
ScreenReader Name: NVDA
MAS Number :MAS1.3.1,MAS4.1.2

**Steps to Reproduce :**
1.	Install the app Azure Data Studio.
2.	Connect to the Database->Go to file->New Notebook.
3.	Click on “Code” button and write query in the box for example SELECT * FROM sys.traces;
4.	Select query->Click on the Run command.
5.	Start NVDA.
6.	Navigate to table.
7.	Observe that screen reader doesn’t announce the table name .

**Actual:**
While navigating through table, screen  reader doesn’t announce the table name.
**Same issue observed with JAWS.**
**Similar issue observed with following pages :
1.MapDataSrcObjects :** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.

**Expected:**
Screen reader should announce the table name.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:**
If user wants to navigate through only table it won't understand which table they are navigating.

**MAS Reference:
MAS4.1.2-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=1a1d7dfd-3dfd-4394-a8af-81fa7c1cf482

**Attachment for Reference:**
[5615_[Accessibility]A11y_ Azure Data Studio_ NotebookQueryTable_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3215611/5615_.Accessibility.A11y_.Azure.Data.Studio_.NotebookQueryTable_ScreenReader.pptx)
	
Does this issue occur when all extensions are disabled?: Yes
